src/main/java/org/sagebionetworks/web/client/widget/evaluation/AdministerEvaluationsListViewImpl.java

```diff
@@ -54,4 +60,20 @@ public Widget asWidget() {
 	public void add(IsWidget w) {
 		widgetsContainer.add(w);
 	}
+
+	@Override
+	public void addReactComponent(Evaluation evaluation, EvaluationCardProps props) {
+		Div container = new Div(){
```

Comment:
This is a great idea.  What do you think about creating something like a ReactComponentDiv that we can use for any Div container being used to render react components (it would have this unmount functionality on unload, and potentially any other react-specific handling)?